src/CommonITILObject.php

```diff
@@ -7783,11 +7783,13 @@ class_exists($validation_class ?? '') && $params['with_validations']
                     'item' => $item,
                     '_is_image' => false,
                 ];
-                $docpath = GLPI_DOC_DIR . "/" . $item['filepath'];
-                $is_image = Document::isImage($docpath);
-                if ($is_image) {
-                    $doc_entry['_is_image'] = true;
-                    $doc_entry['_size'] = getimagesize($docpath);
+                if (!empty($item['filepath'])) {
```

Comment:
I guess it would be better to check for file existence directly in `Document::isImage()` which can be used elsewhere.